Add introduction blog post by anisha (#509)
debezium/debezium.github.io: blog/2020-10-27-hello-debezium.adoc

= Hello Debezium!
anmohant
:awestruct-tags: [ community, news ]
:awestruct-layout: blog-post

Hello everyone, my name is Anisha Mohanty and I recently joined Red Hat and the Debezium team.

> I started my journey with Redhat in April 2020 after completing my graduation. I was introduced to open source in my early college days, but I wasn't aware of how organizations work and wanted to get the essence of open source ethics and values. That is something that I am fascinated to learn as I joined Redhat.

Reviewer: ```suggestion
I started my journey with Red Hat in April 2020 after completing my graduation. I was introduced to open source in my early college days, but I wasn't aware of how organizations work and wanted to get the essence of open source ethics and values. That is something that I am fascinated to learn as I joined Red Hat.
```

> My work started under the Data Virtualization team with Teiid and then under the https://graphqlcrud.org/[GRAPHQLCRUD] project which is a standard for the generic query interface on top of GraphQL. The project has started well and is in great shape right now. We have successfully added CRUD capabilities, paging, and filtering specifications.

Reviewer: ```suggestion
My work started under the Data Virtualization team with Teiid and then under the https://graphqlcrud.org/[GRAPHQLCRUD] project which is a standard for a generic query interface on top of GraphQL. The project has started well and is in great shape right now. We have successfully added CRUD capabilities, paging, and filtering specifications.
```

Not 100% sure; "a" seems better?

> +++<!-- more -->+++

Reviewer: This controls how much is shown on the blog front page. Perhaps pull it up by one paragraph?

Author: Yes. Will be better.

> Coming to Debezium, I first heard about it as some DV members started contributing here, well back then it was a completely new thing for me. I started exploring more, and it was not long when I had my first interaction with Gunnar and Jiri. With a warm welcome and great team here, I am really excited to work for Debezium Community.

Reviewer: ```suggestion
Coming to Debezium, I first heard about it as some DV members started contributing here, well back then it was a completely new thing for me. I started exploring more, and it was not long when I had my first interaction with Gunnar and Jiri. With a warm welcome and great team here, I am really excited to work with the Debezium Community.
```
